.goreleaser.yml

```diff
@@ -8,8 +8,6 @@ builds:
     goarch:
       - amd64
       - arm64
-      - s390x
-      - ppc64le
```

Comment:
we can still build and upload the binaries to the release even if we can't build the images right now

Reply:
Is there much value in doing that? A user may see that and think that we support those architectures and then the docker image doesn't work. They would have to build anyway.